models/intel/human-pose-estimation-0002/description/human-pose-estimation-0002.md

```diff
@@ -0,0 +1,44 @@
+# human-pose-estimation-0002
+
+## Use Case and High-Level Description
+
+This is a multi-person 2D pose estimation network based on the EfficientHRNet approach.
+For every person in an image, the network detects a human pose: a body skeleton consisting of keypoints and connections between them.
+The pose may contain up to 17 keypoints: ears, eyes, nose, shoulders, elbows, wrists, hips, knees, and ankles.
+
+## Example
+
+![](./human-pose-estimation-0002.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Average Precision (AP)          | 44.4%                                     |
+| GFlops                          | 5.9393                                    |
+| MParams                         | 8.1504                                    |
+| Source framework                | PyTorch*                                  |
+
+Average Precision metric described in [COCO Keypoint Evaluation site](https://cocodataset.org/#keypoints-eval).
+
+## Performance
```

Comment:
please remove empty section (see [PR-1749](https://github.com/openvinotoolkit/open_model_zoo/pull/1749)) for example